app/components/Header/index.js

```diff
@@ -39,7 +39,7 @@ function Header(props) {
   return (
     <StyledHeader {...props} data-testid="header">
       <Logo alt="logo" src={logo} />
-      <Title type="heading" id="wednesday_solutions" />
+      <Title type="heading" id="itunesmusic" />
```

Comment:
Refer this for variable nomenclature we follow
https://playbook.wednesday.is/engineering-playbook/the-wednesday-way/react

Reply:
Thanks, will follow that from now onwards